deps/build.jl

```diff
@@ -15,7 +15,9 @@ function find_spinnaker()
     libspinnaker = "libSpinnaker_C.so"
     libspinvideo = "libSpinVideo_C.so"
   else
```

Comment:
I'd make this an `elseif Sys.isapple()` as the user _could_ be trying FreeBSD, so leave the else in with a modified "Linux, Windows and MacOS" message